src/core/graph.js

```diff
@@ -1212,3 +1212,73 @@ export function edgeToParts(
   const timestampMs = edge.timestampMs;
   return {addressParts, srcParts, dstParts, timestampMs};
 }
+
+export function compareGraphs(
+  firstGraph: Graph,
+  secondGraph: Graph
+): {|
+  +isEqual: boolean,
+  +uniqueNodesInFirst: Node[],
+  +uniqueNodesInSecond: Node[],
+  +nodeTuplesWithDifferences: [Node, Node][],
+  +uniqueEdgesInFirst: Edge[],
+  +uniqueEdgesInSecond: Edge[],
+  +edgeTuplesWithDifferences: [Edge, Edge][],
+|} {
```

Comment:
I recommend pulling this type out as an exported `GraphComparison`, it's helpful for complex types like this so we can explicitly say (e.g.) that a `printComparisonResult` function takes a `GraphComparison` as an argument.

Reply:
done